api/rest/client.go

```diff
@@ -18,6 +18,7 @@ import (
 
 type Client struct {
 	baseURL     *url.URL
+	apiURL      *url.URL
```

Comment:
The cloud endpoint is exposed via the `api.circleci.com` subdomain and as such warranted an additional field on the rest client.

Reply:
nit: This probably warrants a comment in the code to make it less confusing